content/rs/concepts/high-availability/rack-zone-awareness.md

```diff
@@ -5,61 +5,53 @@ weight: $weight
-the slaves are much more likely to be available because they reside on
-different racks.
-
-There is no limitation on the number of racks-zones per cluster; each
-node can belong to a different rack or multiple nodes can belong to the
+Rack-zone awareness is a Redis Enterprise feature that helps to ensure high-availability in the event of a rack or zone failure.
+
+When you enable rack-zone awareness in a cluster, you assign
+a rack-zone ID to each node. This ID is used to map the node to a
+physical rack or logical zone. The cluster can then ensure that master shards, corresponding slave shards, and associated endpoints are placed on nodes in different racks/zones.
+
+In the event of a rack or zone failure, the slaves and endpoints in the remaining racks/zones will be promoted. This ensures high availability when a rack or zone fails.
+
+There is no limitation on the number of rack-zones per cluster; each
+node can belong to a different rack, or multiple nodes can belong to the
 same rack.
 
 Rack-zone awareness affects various cluster, node and database-related
-actions, such as node rebalancing, node removal, node replacement, shard
-and endpoint migration, and database failover.
+actions, such as node rebalancing, node removal, node replacement, shard and endpoint migration, and database failover.
 
-To benefit from rack-zone awareness, you need to configure it at the
-cluster, node and database levels.
+## Cluster and node configuration {#clusternode-configuration}
 
-## Cluster/node configuration {#clusternode-configuration}
+To enable rack-zone awareness, you need to configure it at the
+cluster, node, and database levels.
 
-In order for the cluster to support rack-zone awareness, you need to
-enable it when initially creating the cluster. Afterwards, every time
-you add a new node to the cluster you need to define a "rack-zone ID"
-for the node.
+First, enable rack-zone awareness when you initially create the cluster.
+
+Now, every time you add a new node to the cluster, define a **rack-zone ID** for the node.
 
 The rack-zone ID must comply with the following rules:
 
-- Length: up to 63 characters.
-- Characters: only letters, digits and hyphen ('-') are allowed.
-- Start with a letter and end with a letter or a digit.
-- Case insensitive: note that uppercase and lowercase letter are
-    treated exactly the same.
+- Maximum length of 63 characters.
+- Characters consist of letters, digits, or hyphens ('-').
+- ID starts with a letter and ends with a letter or a digit.
+
+Note: rack-zone IDs are **case-insensitive** (uppercase and lowercase letter are treated as the same).
 
-**Note**: The rack-zone ID is case-insensitive, i.e. uppercase and
-lowercase letters are treated exactly the same.
+### Node layout
+
+Recall that the recommended minimum number of nodes in a RS deployment is three. For rack-zone awareness, these three nodes must be distributed across three *distinct* racks or zones.
```

Comment:
It's not for "rack-zone awareness", that the three nodes should be distributed across 3 racks, it's for high-availability. I think we need to explain here why 3 racks / zones are required here / link to the relevant page